src/ng/directive/ngEventDirs.js

```diff
@@ -81,7 +81,7 @@ forEach(
  * @name ngDblclick
  *
  * @description
- * The `ngDblclick` directive allows you to specify custom behavior on a dblclick event.
+ * The `ngDblclick` directive allows you to specify custom behavior on a dblclick event. Unless used with "ng-click" then ng-click not fires.
```

Comment:
Sorry, I don't understand what this means. Which event does not fire when dblclick is used with click?